sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -99,16 +107,39 @@ def test_queue_by_queue_client_conn_str_receive_handler_peeklock(self, servicebu
             with sb_client.get_queue_sender(servicebus_queue.name) as sender:
                 for i in range(10):
                     message = Message("Handler message no. {}".format(i))
-                    message.enqueue_sequence_number = i
+                    message.properties = {'key': 'value'}
```

Comment:
Silly nitpick: test for set/unset of session_id? (didn't see a complement to this in the test_sessions subsection)

Reply:
unlike annotations, `session_id` is an instance variable on MessageProperties which is rather safe to set/unset.
but sanity check is always welcome, tests updated.